api.php

```diff
@@ -0,0 +1,52 @@
+<?php
+$status = 200;
+ini_set('xdebug.max_nesting_level', 10000);
```

Comment:
I've added JSONP support already, but not sure we need separate endpoint for this... Adding docs is a good idea though.